server/datastore/psql/delete_maintenance_request.go

```diff
@@ -1,6 +1,6 @@
 package psql
 
-
+//Deletes the row of the maintenance request in the database with the specified id endpoint
```

Comment:
Golang convention is to start with the name of the function, so `DeleteMaintenanceRequest...`